stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -206,6 +221,14 @@
 ;;   for the given reward cycle
 (define-map aggregate-public-keys uint (buff 33))
 
+;; Data var used to track signer set for current cycle
+(define-data-var current-signer-set (list 4000 (buff 33)) (list ))
+
+;; Map that tracks the signing-key & cycle to a list of key-ids
+(define-map reward-cycle-signing-key-ids 
+    {cycle: uint, signer: (buff 33)}
+    {key-ids: (list 4000 uint)})
```

Comment:
Can you add a comment explaining what a `key-id` is supposed to be, and why it's necessary in addition to the `signer-address` in `reward-cycle-signing-keys` and `signer` in `reward-cycle-signing-key-ids`?  Also, why have both the signing key and the address?